EIPS/eip-5005.md

```diff
@@ -0,0 +1,225 @@
+---
+eip: 5005
+title: Zodiac
+description: A composable design philosophy for DAOs.
+author: Auryn Macmillan (@auryn-macmillan), Kei Kreutler (@keikreutler)
+discussions-to: https://ethereum-magicians.org/t/eip-zodiac-a-composable-design-philosophy-for-daos/8963
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-04-14
+requires: 165
+---
+
+## Abstract
+Zodiac is a philosophy and open standard for composable and interoperable DAO tooling. The key insight is that by separating account (thing that holds tokens, controls systems, and is referenced by others) from the logic that controls it, and making the account agnostic to the logic that controls it, we can create a DAO ecosystem that is much more composable and interoperable, along with being compatible with the vast majority of existing DAO tooling. In short, Zodiac encourages DAOs to use an "avatar" contract (like the Gnosis Safe) as their account, and to then control that avatar with just about any combination of DAO tools and frameworks (Aragon, Colony, Compound/OZ Governor, DAOStack, Moloch, Orca, Tribute, etc).
```

Comment:
I think this abstract is a little heavy on motivation, and a little light on technical details. Perhaps something like:

```suggestion
Zodiac is a philosophy and open standard for composable and interoperable DAO tooling. A Zodiac-compatible DAO separates the account taking actions/holding tokens (known as the "avatar") and the authorization logic into two separate contracts. This standard defines the `IAvatar` interface, to be implemented by avatar contracts, while the authorization logic can be implemented with any combination of DAO tools and frameworks (Aragon, Colony, Compound/OZ Governor, DAOStack, Moloch, Orca, Tribute, etc).
```

Reply:
Thanks! This is a really nice simplification.